Product/Production/Adapters/General/CONNECTAdminGUI/src/main/webapp/certificateManager.xhtml

```diff
@@ -722,7 +724,12 @@
                                                         icon="ui-icon-alert" />
                                                 </p:commandButton>
                                                 <p:commandButton styleClass="btn btn-primary" icon="ui-icon-seek-end"
-                                                    value="Backup and Replace" />
+                                                    value="Complete" immediate="true" update="@form"
+                                                    action="#{certificateBean.completeImportWizard()}">
+                                                    <p:confirm header="Complete Import Wizard"
+                                                        message="Completing the import wizard will create backup and new, and delete temporary (KeyStore and TrustStore). Do you wanted to continue?"
```

Comment:
Continue with creating backup of existing KeyStore and TrustStore and saving newly constructed KeyStore and TrustStore?